src/main/java/nextstep/ladder/domain/Direction.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.ladder.domain;
+
+import java.util.Arrays;
+
+public enum Direction {
+    LEFT(-1),
+    STAY(0),
+    RIGHT(1);
+
+    private final int value;
+
+    Direction(int value) {
+        this.value = value;
+    }
+
+    public static Direction of(int value) {
+        return Arrays.stream(Direction.values())
+                .filter(direction -> direction.getValue() == value)
```

Comment:
```suggestion
                .filter(direction -> direction.sameValue(value))
```

값을 꺼내오기 보다는 객체에게 물어보면 어떨까요?